### PR TITLE
Update tutorial.rst

### PR DESCRIPTION
**Pull request by brianzhang01**
Make the Tables example consistent, since the edge between node 4
and node 1 should extend throughout the entire chromosome. Also
make some small wording and punctuation changes.

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -218,7 +218,7 @@ input, and returns a copy in which all singleton sites have been removed.
 The approach is very simple: we get a copy of the underlying
 table data in a :class:`.TableCollection` object, and first clear the
 site and mutation tables. We then consider each site in turn,
-and if the allele frequency of
```
Comment (brianzhang01) [inline docs/tutorial.rst]: I interpret "allele frequency" as a probability value between 0 and 1

**Comment by jeromekelleher** [comment]
Thanks for this @brianzhang01, these are great updates. It looks like there's some slight issues git-wise. Can you rebase this against upstream/master please (see [here](https://stdpopsim.readthedocs.io/en/latest/development.html#github-workflow) for some guidance on a similar project). If you're not comfortable with this, just let me know, I'm happy to do the rebasing.

**Comment by brianzhang01** [comment]
The detailed writeup on `git rebase` was useful, thanks. We can try the merge again.

**Comment by jeromekelleher** [comment]
Perfect, thanks @brianzhang01!

**Pull request merged**